packages/twenty-server/src/engine/metadata-modules/field-metadata/field-metadata.service.ts

```diff
@@ -77,6 +78,7 @@ export class FieldMetadataService extends TypeOrmQueryService<FieldMetadataEntit
     private readonly dataSourceService: DataSourceService,
     private readonly typeORMService: TypeORMService,
     private readonly workspaceMetadataVersionService: WorkspaceMetadataVersionService,
+    private readonly twentyORMGlobalManager: TwentyORMGlobalManager,
```

Comment:
Hello @lucasbordeau 
do you mean we should use `TwentyORMManager` instead of `TwentyORMGlobalManager` or do want to use `TwentyORMGlobalManager` without using the service's constructor?

if you are referring to the second option, we would need to do this
![image](https://github.com/user-attachments/assets/5b4d275c-5516-49cb-99fa-d2c480376e8b)

if you are referring to the first option, the `TwentyORMManager` class is different from `TwentyORMGlobalManager`, we don't have the `getRepositoryForWorkspace` method, only `getRepository` we are not sure how this can affect the feature

Reply:
Ok